setup.cfg

```diff
@@ -0,0 +1,76 @@
+[metadata]
+name = imagecraft
+version = attr: imagecraft.__version__
```

Comment:
My python-brain does not comprehend this version number handling..!

Reply:
I discovered this is the idiomatic way to use the `__version__` attribute defined/computed in the package. See https://packaging.python.org/en/latest/guides/single-sourcing-package-version/ (specifically the `Note`).